include/rocksdb/cloud/cloud_env_options.h

```diff
@@ -60,12 +60,48 @@ enum class CloudRequestOpType {
 using CloudRequestCallback =
     std::function<void(CloudRequestOpType, uint64_t, uint64_t, bool)>;
 
+class BucketOptions {
+private:
+  std::string bucket_;
+  std::string prefix_;
+  std::string object_;
+  std::string region_;
+  std::string name_;
```

Comment:
can you comment on all the various options? `bucket_`, `prefix_` and `name_` are fairly confusing. Also, mention that default value of `prefix_` is `rockset`.